### ISSUE TITLE
Can't enter multiple lines in interactive window

### ISSUE THREAD

**Issue by mousetraps** [Microsoft/nodejstools#1300]
![image](https://cloud.githubusercontent.com/assets/762848/19053849/8e2851d8-8971-11e6-8430-99283ba51657.png)

##### Steps to Reproduce
1. type `function () { <enter>` in IW

**Comment by mjbvz** [comment]
Yes, after ripping out the static analysis engine and removing code analysis from the interactive window, we no longer support automatically entering multiline statements in the interactive window. You now have to press `CTRL + SHIFT + RETURN` to enter a new line. Automatic indention of multiline statements in the interactive window is also not supported any longer.

**Comment by mjbvz** [comment]
Reopening to track resolving this eventually this it is a known regression. This may be something we investigate further with #1090 in dev15.

**Comment by paulvanbrenk** [comment]
This requires interaction with th JavaScript LS, to figure out if [this](https://github.com/Microsoft/nodejstools/blob/0ca4b431741cd2b3a0e4c099b0b1321d1385f319/Nodejs/Product/Nodejs/Repl/NodejsReplEvaluator.cs#L118) method should retur `true`.
